compiler/semtempl.nim

```diff
@@ -197,6 +197,13 @@ proc addLocalDecl(c: var TemplCtx, n: var PNode, k: TSymKind) =
     else:
       replaceIdentBySym(c.c, n, ident)
   else:
+    if (n.kind == nkPragmaExpr and sonsLen(n) >= 2 and n.sons[1].kind == nkPragma):
+      let pragmaNode = n.sons[1]
+      let numPragmas = sonsLen(pragmaNode)
+      for i in 0 .. numPragmas-1:
```

Comment:
This loop should start at ``1``.

Reply:
```suggestion

for i in 1..<pragmaNode.len:

```

Reply:
Loop needs to start at `0` so the first pragma isn't ignored but I used your suggestion iterating through.

Reply:
ah right, I misread, my bad.